src/cuter_pp.erl

```diff
@@ -725,7 +725,11 @@ pp_arguments(Args) ->
 pp_argument(X) ->
   case cuter_lib:is_lambda(X) of
     true  -> pp_lambda(X);
-    false -> io_lib:format("~p", [X])
```

Comment:
This code can be simplified to:
```erlang
false -> io_lib:format("~p", [cuter_lib:compile_lambda(X)])
```
which will avoid duplication of similar-looking code and will also allow to remove the export of the `default_any_value/0` function from `cuter_lib`.